coretex/entities/task_run/parameter/utils.py

```diff
@@ -65,3 +67,36 @@ def validateRangeStructure(name: str, value: Dict[str, Any], required: bool) ->
         return False, "Range parameter does not support float values"
 
     return True, None
+
+def getParamTypeByValueType(value: Any, name: str) -> ParameterType:
+    if isinstance(value, int):
```

Comment:
Perform the `bool` instance check before `int` check. In python `bool` is a subclass of `int`.